### PR TITLE
getDisplayMedia available in Firefox 66 + clarify Chrome/Edge versions

### PR DESCRIPTION
**Pull request by jan-ivar**
In [Firefox 66](https://bugzilla.mozilla.org/show_bug.cgi?id=1321221), [Edge 17](https://blogs.windows.com/msedgedev/2018/05/02/bringing-screen-capture-to-microsoft-edge-media-capture-api/), [Chrome 72](https://www.chromestatus.com/feature/6744724455030784). @a2sheppy PTAL.

### CODE REVIEW

File: api/MediaDevices.json [hunk 1]
```diff
@@ -228,27 +228,70 @@
...
+              "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code>."
             },
             "edge_mobile": {
               "version_added": true
             },
-            "firefox": {
-              "version_added": false,
-              "notes": "Since Firefox 33 you can capture screen data using <code><a href='https://developer.mozilla.org/docs/Web/API/MediaDevices/getUserMedia'>getUserMedia()</a></code>, with a <code>video</code> constraint called <code>mediaSource</code>."
-            },
             "firefox_android": {
```
Comment (Elchi3) [inline api/MediaDevices.json]: You have` firefox_android` twice.

File: api/MediaDevices.json [hunk 2]
```diff
@@ -228,27 +228,70 @@
...
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "Experimental Web Platform features",
+                    "value_to_set": "Enabled"
+                  }
+                ],
+                "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
+              }
+            ],
```
Comment (jpmedley) [inline api/MediaDevices.json]: As stated in the [Chrome Status entry](https://www.chromestatus.com/features/6744724455030784) this is NOT yet in Android. I've confirmed this with the feature owners.

Comment (jpmedley) [reply api/MediaDevices.json]: I also suspect this note doesn't apply to Android. Please don't merge until I've confirmed.

Comment (jan-ivar) [reply api/MediaDevices.json]: Good catch. I've updated it. I also added a note about Firefox for android.

File: api/MediaDevices.json [hunk 3]
```diff
@@ -228,26 +228,46 @@
...
             "chrome_android": {
-              "version_added": "70",
-              "notes": "Available as a member of <code>Navigator</code> instead of <code>MediaDevices</code> in Chrome 70 and 71."
+              "version_added": false
```
Comment (jpmedley) [inline api/MediaDevices.json]: This also applies to `webview_android`.

**Pull request merged**